ex1/ex1.c

```diff
@@ -2,13 +2,29 @@
 // (e.g., x) and set its value to something (e.g., 100). What value is the variable in the child process?
 // What happens to the variable when both the child and parent change the value of x?
 
+// The variables change independently!
+
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
 
 int main(void)
 {
+    int x = 100;
+    int child = fork();
     // Your code here
+    if (child < 0){
+        printf("FORK FAILED.\n");
+        exit(1);
+    } else if (child == 0) {
+        printf("Child (before): %i\n", x);
+        x = 42;
+        printf("Child (after): %i\n\n", x); // If you remove the second /n it changes the order. Why?
```

Comment:
It looks like you're using \n twice, have you considered thats changing the order?